tests/robot-tests/tests/admin/analyst/analyst_absence_permissions.robot

```diff
@@ -13,8 +13,7 @@ Validate Analyst1 can see correct themes and topics
     user waits until page contains accordion section    Pupil absence in schools in England    60
 
     user checks select contains option    id:publicationsReleases-themeTopic-themeId    Pupils and schools
-    user checks select contains x options    id:publicationsReleases-themeTopic-topicId    2
-    user checks select contains option    id:publicationsReleases-themeTopic-topicId    Exclusions
+    user checks select contains x options    id:publicationsReleases-themeTopic-topicId    1
```

Comment:
This error was due to analyst1's contributor permission for the Exclusions release being removed. I've readded it and this now passes, so this change can be reverted.